docs/sdk/server-side-sdks/go/go-gettingstarted.md

```diff
@@ -37,7 +37,7 @@ func main() {
 		DisableCustomEventLogging:    false,
 	}
 
-	client, err := devcycle.NewClient(sdkKey, &options)
+	devcycleClient, err := devcycle.NewClient(sdkKey, &options)
```

Comment:
I don't think this is needed. It's a bit verbose

Reply:
its consistent with the other examples / docs